README.md

```diff
@@ -0,0 +1,123 @@
+<section align="center">
+
+# ``skglm``
+
+## A fast :zap: and modular :hammer_and_pick: scikit-learn replacement for sparse GLMs
+
+</section>
+
+![build](https://github.com/scikit-learn-contrib/skglm/workflows/pytest/badge.svg)
+![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)
+![Downloads](https://pepy.tech/badge/skglm/month)
+![PyPI version](https://badge.fury.io/py/skglm.svg)
+
+
+``skglm`` is a Python package that offers **fast estimators** for sparse Generalized Linear Models (GLMs) that are **100% compatible with ``scikit-learn``**. It is **highly flexible** and supports a wide range of GLMs. You get to choose from ``skglm``'s already-made estimators or **customize your own** by combing the available datafits and penalty.
+
+Excited to have a tour on ``skglm`` [documentation](https://contrib.scikit-learn.org/skglm/) :memo:?
+
+# Why ``skglm``?
+
+``skglm`` is specifically conceived to solve sparse GLMs.
+It supports many missing models in ``scikit-learn`` and ensures high performance.
+There are several reasons to opt for ``skglm`` among which:
+
+|  |  |
+| ----- | -------------- |
+| **Speed** :zap: | Fast solvers able to tackle large datasets, either dense or sparse, with millions of features **up to 100 times faster** than ``scikit-learn``|
+| **Modularity** :hammer_and_pick: | User-friendly API than enables **composing custom estimators** with any combination of its existing datafits and penalties |
+| **Extensibility** :arrow_up_down: | Flexible design that makes it **simple and easy to implement new datafits and penalties**, a matter of few lines of code
+| **Compatibility** :electric_plug: | Estimators **fully compatible with the ``scikit-learn`` API** and drop-in replacements of its GLM estimators
+|  |  |
+
+
+# Get started with ``skglm``
+
+## Installing ``skglm``
+
+``skglm`` is available on PyPi. Run the following command to get the latest version of the package
+
+```shell
+pip install -U skglm
+```
+
+It is also available on Conda _(not yet, but very soon...)_ and can be installed via the command
+
+```shell
+conda install skglm
+```
+
+## Trying ``skglm``
+
+Once you installed ``skglm``, you can run the following code snippet to fit an MCP Regression model on a toy dataset
+
+```python
+# import model to fit
+from skglm.estimators import MCPRegression
+# import util to create a toy dataset
+from skglm.utils import make_correlated_data
+
+# generate a toy dataset
+X, y, _ = make_correlated_data(n_samples=10, n_features=100)
+
+estimator = MCPRegression()
+estimator.fit(X, y)
+
+# print R²
+print(estimator.score(X, y))
+```
+You can refer to the documentation to explore the list of ``skglm``'s already-made estimators. 
+
+Didn't find one that suits you :monocle_face:, you still can compose your own.
+Here is a code snippet that fits the previous estimator with a Huber loss instead.
```

Comment:
A bit confusing. I'd say `Here is a code snippet that fits a MCP-regularized problem with Huber loss`.